### PR TITLE
hslSegmentFormat and hlsVideoSegmentFormat android support

### PR DESCRIPTION
**Pull request by tomas-lejdung**
https://github.com/react-native-google-cast/react-native-google-cast/issues/278#issuecomment-836870166

Adds support for HLSSegmentFormat and HLSVideoSegmentFormat for android.

https://developers.google.com/android/reference/com/google/android/gms/cast/HlsSegmentFormat
https://developers.google.com/android/reference/com/google/android/gms/cast/HlsVideoSegmentFormat#MPEG2_TS

### CODE REVIEW

File: android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java
Comment (petrbela) [inline android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java:10]: Since these are abbreviations already, I'd just use the name as the string value, e.g. `AAC`, `AC-3`, `E-AC-3`, `fMP4`, MP3`, `TS`, `TS-AAC` (is this even a thing?), and `MPEG-TS`.

File: src/types/MediaInfo.ts
Comment (petrbela) [inline src/types/MediaInfo.ts:35]: Maybe add `*Android only*` to the doc?

File: android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java
Comment (tomas-lejdung) [reply android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java:10]: What do you mean? That the strings in lowercase should be in uppercase instead? I simply used the string values of the constants.

Comment (petrbela) [reply android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java:10]: Yeah, I'd make the strings uppercase since they're like proper names, not actual words.

Comment (tomas-lejdung) [reply android/src/main/java/com/reactnative/googlecast/types/RNGCMediaHLSSegmentFormat.java:10]: These values should map to the string enums defined in the typescript files, right?

**Review by petrbela — APPROVED** [review]

**Comment by tomas-lejdung** [comment]
FYI I cannot merge this as I do not have write access :)

**Comment by arnelisk** [comment]
Are there any plans to release this update?

**Pull request merged**
